ui/de-webapp/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+multipart.max-file-size=10000000
+multipart.max-request-size=10000000
```

Comment:
Isn't this stuff what `de-application.yaml` does? Or is this some sort of better approach to that (which, hopefully, we can still override in prod)?